### DISCUSSION TITLE
Version 0.6.0 Beta 5

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1722] [Announcements]
## **Application download links for each operating system are available below in the Assets section.**

**WARNING: Breaking Changes.**  This version changes the DMR decoder channel configuration's channel to frequency mapping format.  Previously, the user specified each Logical Slot Number (LSN) and the corresponding frequency and this required duplicate entries, one for each timeslot.  This has been updated to use the Logical Channel Number (LCN) and you only have to input one map entry for each repeater LCN.  The LCN and Frequency values can be found in the Radio Reference details for the DMR system. 

If you wish to continue using versions prior to 0.6.0 Beta 2, please make a backup copy of your playlist before you modify your current playlist to change the DMR channel to frequency maps.  That way you can switch between playlists.

## 1. Features:
* SDRPlay RSP tuner support  (API versions 3.07 - 3.11)
* Airspy Discovery HF+ support
* RTL2832 Support for R828D (aka rtl-sdr Blog V4) and FC0013 Tuners
* AM & NBFM Adaptive Squelch & AM Decoder Enhancements
* Digital Coded Squelch (DCS) aux decoder available with NBFM decoder
* LTR, LTR-Net & Passport Decoder enhancements
* rdio-scanner call broadcasting
* Update to OpenJDK 20 & Gradle 8.2

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. What's Changed
* #968 Playlist Alias Editor - Clear Overlap Error Warning Once Resolved by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1712
* #967 Multi-Alias Editor Supports Changing Record Audio Status by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1713
* #732 P25 Phase 2 - FROM ID Updated For Call Event Logging by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1714
* #1715 rtl-sdr Blog V4 Tuner Issue When Using V4 Dongle With Other R820T Tuners by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1716
* #1681 Audio Recorder Mixing Up System and Site When Decoding Multiple Systems  by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1717
* #1619 JavaFX Windows Show Minimized by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1719
* #1720 Updates version to 0.6.0 Beta 5 by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1721

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.6.0-beta-4...v0.6.0-beta-5

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/v0.6.0-beta-5'>Version 0.6.0 Beta 5</a>.</em>

**Comment by Ertcom** [comment]
Thank you. I'm not sure if this is a bug or my lack of setup understanding........The call event log is not showing any data in the "TO" column. I'm trying to see calls to an APCO25 talkgroup. It is present in the live screen but not in the logs.

TIMESTAMP,DURATION_MS,PROTOCOL,EVENT,FROM,TO,CHANNEL_NUMBER,FREQUENCY,TIMESLOT,DETAILS
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
"2023:11:20:10:50:13","","APCO-25","Encrypted Patch Call","xxxxxx","","10-112 TS1","421.837500","","CHANNEL START REJECTED - PHASE 2 CHANNEL GRANT PRI0 ENCRYPTED CIRCUIT"
